src/sidebar/components/test/HelpPanel-test.js

```diff
@@ -61,19 +61,21 @@ describe('HelpPanel', () => {
 
     it('should show navigation link to versionInfo sub-panel', () => {
       const wrapper = createComponent();
-      const link = wrapper.find('.HelpPanel__sub-panel-navigation-button');
+      const button = wrapper.find('HelpPanelNavigationButton');
 
-      assert.include(link.text(), 'About this version');
+      assert.include(button.text(), 'About this version');
     });
 
-    it('should switch to versionInfo sub-panel when footer link clicked', () => {
+    it('should switch to versionInfo sub-panel when navigation button clicked', async () => {
       const wrapper = createComponent();
-      wrapper.find('.HelpPanel__sub-panel-navigation-button').simulate('click');
+      act(() => {
```

Comment:
The [`invoke`](https://github.com/enzymejs/enzyme/blob/master/docs/api/ReactWrapper/invoke.md) wrapper method could be used to shorten this sequence:

```js
// nb. No need for `act` or `wrapper.update` if using this
wrapper.find('LinkButton').invoke('onClick')();
```

Reply:
Sounds promising (I literally did not know about `invoke`), but in this case there is a special need. The callback needs a real Event object because it invokes `preventDefault` on it, so the way this test is structured is to provide that `Event` object.